github/gists.go

```diff
@@ -52,6 +52,30 @@ func (g GistFile) String() string {
 	return Stringify(g)
 }
 
+// GistCommit represents a commit on a gist.
+type GistCommit struct {
```

Comment:
Based on my reading of https://developer.github.com/v3/gists/#list-gist-commits, this struct seems to be missing the URL field:

```Go
URL *string `json:"url,omitempty"`
```